weave/__init__.py

```diff
@@ -24,6 +24,7 @@
 from . import wandb_api as _wandb_api
 
 from .core_types import *
+from .monitoring2 import *
```

Comment:
why `monitoring2` instead of extending existing monitoring?

Reply:
It will be `monitoring`, this is just namespaced for dev